packages/11ty/_includes/components/accordion/global-controls.js

```diff
@@ -0,0 +1,20 @@
+const { html } = require('~lib/common-tags')
+
+/**
+ * @param {Object} eleventyConfig
+ * @returns Global Accordion Controls Shortcode
+ */
+module.exports = function(eleventyConfig) {
+  const { collapseText, expandText } = eleventyConfig.globalData.config.accordion.globalControls
```

Comment:
Naming this field `labels` (see code suggestion below) or perhaps the individual fields something such as `collabseButtonLabel` may be more explicit?
```suggestion
  const { labels: { collapse, expand } } = eleventyConfig.globalData.config.accordion.globalControls
```